tests/test_model/test_updating_time_series.py

```diff
@@ -0,0 +1,395 @@
+import tempfile
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+
+from ispypsa.model import build_pypsa_network, update_network_timeseries
+
+
+def test_expand_then_operate():
+    """
+    Test expanding the network with two snapshots per year for two one year investment
```

Comment:
```suggestion
    Test expanding the network with two snapshots per year for two single-year investment
```
(For clarity - lots of word numbers makes my eyes go funny)